### DISCUSSION TITLE
Newest version: breaking change in config || a bug?

### DISCUSSION THREAD

**Discussion by ypakhomenko** [cloudprober/cloudprober#693] [Q&A] [answered]
Hi, 
I am trying to move from:
/usr/local/bin/cloudprober -version
**v0.12.6**
to:
./cloudprober -version
**v0.13.2-16-g6d06eb93**

The old version perfectly worked, the new version reports a problem with the same config (on the same VM):

level=WARN **source=probes/http/http.go:284** msg="Get \"https://cloudprober-target.caas-t07.telekom.de:443/\": **context deadline exceeded**" system=cloudprober probe=CaaS_t07_GCP_LZ_Target target=cloudprober-target.caas-t07.telekom.de url=https://cloudprober-target.caas-t07.telekom.de:443/

And the target cannot be reached (timeout).
Probably I missed some breaking change in the config || can it be a bug?

The config:

```
surfacer {
        type: STACKDRIVER
}
surfacer {
  type: PROMETHEUS
}

probe {
        name: "dialog-telekom.de"
        type: HTTP
        targets {
                host_names: "www.dialog-telekom.de"
        }
        http_probe {
            protocol: HTTPS
            port:443
            relative_url: "/"
            max_redirects: 0

           tls_config {
              disable_cert_validation: true
           }
        }        
        validator {
              name: "status_code_any"
              http_validator {
                  success_status_codes: "200-399"
              }
        }
       #validator {
       #   name: "simple_re"
       #   regex: "geschaeftskunden"
       #}

        interval_msec: 60000
        timeout_msec: 2000 

}
probe {
        name: "CaaS_t07_GCP_LZ_Target"
        type: HTTP
        targets {
                host_names: "cloudprober-target.caas-t07.telekom.de"
        }
        http_probe {
            protocol: HTTPS
            port:443
            relative_url: "/"
        }        

        interval_msec: 10000
        timeout_msec: 1000 
}
```

Thanks a lot,
Yury

**Comment by manugarg** [comment]
@ypakhomenko I don't see any obvious issues in the config.

Maybe you can try other intermediate versions to see where it breaks:
https://github.com/cloudprober/cloudprober/releases

**Comment by ypakhomenko** [comment, marked as answer]
Hi again and thank you for your quick response,
after some more debugging i must admit, that the problem was in front of the monitor :-(
The difference was that the old version was called as a daemon (with systemctl, with properly configured proxy and no_proxy vars) 
and the latest version was called per hand as #./cloudprober .... and the env was not properly set.
Sorry for confusion. 
Maybe you should introduce a new error message, e.g.: "Wrong user. Remove the user and continue the normal operations" :-)

**Comment by manugarg** [comment]
haha, no worries. Thanks for confirming!
